package.json

```diff
@@ -165,6 +161,7 @@
     "@scalprum/core": "0.0.10",
     "@scalprum/react-core": "0.0.15",
     "@sentry/browser": "^6.1.0",
+    "abortcontroller-polyfill": "^1.7.1",
```

Comment:
Any reason for this dependency? I don't see it used.

Reply:
Inventory is using the abort for stopping some requests and it's not part of dependencies list.

Reply:
Oh. I think we should add it to inventory and remove it from here (in a future PR)